src/intercom/front_end_binding.py

```diff
@@ -55,7 +59,7 @@ def get_binary_search_result(self, request_id):
     def _request_response_listener(self, input_data, request_connection, response_connection):
         serialized_request = pickle.dumps(input_data)
         request_id = generate_task_id(input_data)
-        self.connections[request_connection]['fs'].put(serialized_request, filename="{}".format(request_id))
+        self.connections[request_connection]['fs'].put(serialized_request, filename='{}'.format(request_id))
```

Comment:
```suggestion
        self.connections[request_connection]['fs'].put(serialized_request, filename=request_id)
```

`generate_task_id` always returns a string :-)

Reply:
Pre-commit fixed that one, I didn't even look closer at it :sweat_smile:  => fixed